### DISCUSSION TITLE
Layered obstructions

### DISCUSSION THREAD

**Discussion by zzlcherry** [firemodels/fds#13348] [Q&A]
Warning
Obstructions that use a layered surface with the backing type EXPOSED can only occupy the boundary cells of a mesh when all adjacent mesh cells are one-to-one. This model contains 59 obstructions that violate this requirement.
Will this problem affect the calculation result and how to solve it?

**Comment by mcgratta** [comment]
Could you print out exactly what the warning says so that I can search the code for exactly that text string.

**Reply by zzlcherry** [reply]
Obstructions that use a layered surface with the backing type EXPOSED can only occupy the boundary cells of a mesh when all adjacent mesh cells are one-to-one. This model contains 59 obstructions that violate this requirement.
Press OK to select affected obstructions.

**Reply by mcgratta** [reply]
This appears to be a message from PyroSim. I assume this message is given before you even start the simulation. My guess is that you are changing the mesh size from one mesh to another, and that your obstructions are going to "snap" to different locations on either side of the mesh, making it difficult for FDS to do the heat conduction through the obstruction.

**Reply by zzlcherry** [reply]
ok thank you!
